packages/round-manager/src/features/api/__tests__/application.test.ts

```diff
@@ -103,3 +103,72 @@ describe("getApplicationById", () => {
     consoleErrorSpy.mockClear();
   });
 });
+
+describe("getApplicationsByRoundId", () => {
+  
+  let expectedApplications: GrantApplication[];
+
+  beforeEach(() => {
+    expectedApplications = [makeGrantApplicationData()];
+    // const projectOwners = expectedApplication.project?.owners.map(
```

Comment:
actually i think this can be removed -- RM doesn't need to fetch project owners for the bulk application view; the project owners are only fetched when viewing an individual application's details. so `getApplicationsByRoundId` doesn't need to fetch the info

Reply:
removed in [ab9d830](https://github.com/gitcoinco/grants-round/pull/416/commits/ab9d8304102d22e01ea8b3094d05c07fed093e47)